src/resolvers/uptimeResolvers.ts

```diff
@@ -7,14 +7,40 @@
  */
 
 import { fetchUptimeIssues } from '../api';
+import { fetchVersion } from '../api/taxonomyApi';
+
+const localizedVersionHashTitle = (name: string, language: string) => {
+  switch (language) {
+    case 'nb':
+      return `Du ser på en forhåndsvisning av versjonen "${name}"`;
+    case 'nn':
+      return `Du ser på ei førehandsvisning av versjonen "${name}"`;
```

Comment:
Her blei det en n for masse. Det skulle vore "førehandsvising"

Reply:
Ups, kan ta en hotfix.